src/20_Components/amp-facebook-comments.html

```diff
@@ -0,0 +1,61 @@
+<!--
+  ## Introduction
+
+  The amp-facebook-comments component allows embedding Facebook comments into AMP files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+  <title>amp-facebook-comments</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- Import the amp-facebook-comments component -->
+    <script async custom-element="amp-facebook-comments" src="https://cdn.ampproject.org/v0/amp-facebook-comments-0.1.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-facebook-comments/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+
+    <!-- ## Basic Usage -->
+    <!-- The `amp-facebook-comments` component only requires the href of the comments. -->
+    <amp-facebook-comments width="486" height="657"
+        layout="responsive"
+        data-href="http://www.directlyrics.com/adele-25-complete-album-lyrics-news.html">
+    </amp-facebook-comments>
+
+    <!-- ## Customizing the embed -->
+    <!-- The `amp-facebook-comments` component allows you to customize the embed via several attributes: -->
+
+    <!-- **Example 1: You can set the locale to render the comments in a different language, for example French.** -->
```

Comment:
See comment for amp-facebook-page PR. Please replace "Example X" with a subheading.

Reply:
Done